app/views/organisers/problematic_projects.slim

```diff
@@ -0,0 +1,34 @@
+.page-header
+  h2.text-center.green-text Problematic Projects
+
+= link_to "Back to All projects", dashboard_organisers_path
+.row
+  .col-xs-12
+    table.table.table-striped.table-bordered
+      thead
+        tr
+          th.col-md-4 Mentee
+          th.col-md-4 Mentor
+          th.col-md-2 Project
+          th.col-md-2 GitHub Link
+          th.col-md-2 Missing
+      tbody
+        - @projects.each do |project| 
+          tr
+           td
+             = link_to project.mentee.full_name, mentee_profile_path(project.mentee_id)
+           td
+             = link_to project.mentor.full_name, mentor_profile_path(project.mentor.id)
+           td = project.try(:title)
+           td
+             = link_to url_with_protocol(project.github_link), target: "_blank" do
+               i.fa.fa-github.fa-3
+           td 
+             - if (project.mentee.is_missing == true) and (project.mentor.is_missing == true)
```

Comment:
@pg20 last thing - because is_missing is a boolean you don't need to compare it to true
`project.mentee.is_missing && project.mentor.is_missing` is enough